kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -382,8 +382,7 @@ define([
                         inputWidget,
                         appSpec,
                         spec,
-                        initialParams[spec.id],
-                        filePathParams.layout
```

Comment:
This param was flat out wrong -- it's supposed to be a list of close parameters, but instead it's the widget layout. Removing.

Reply:
:confounded:

Reply:
Is closeParameters needed here to start the field widget? Should that be pulled out of the model?

Reply:
no, it isn't needed -- it would previously have caused an error if there was nothing supplied, but I added in a default so the widget doesn't die if the param is missing